axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -403,4 +415,12 @@ public List<SequenceVersion> updateSequenceVersions(
 
     return sequenceVersionList;
   }
+
+  public void validateSequence(Sequence sequence) throws AxelorException {
+    if (sequence.getPrefixe().startsWith(appBaseService.getAppBase().getDraftPrefix()))
+      throw new AxelorException(
+          appBaseService.getAppBase(),
+          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
+          I18n.get("Sequence prefix cannot contain draft prefix"));
```

Comment:
- [ ] Error messages should be centralized in BaseExceptionMessage.
- [ ] Add French translation
- [ ] Also change the implementation as follows:

```suggestion
    String draftPrefix = getDraftPrefix();
    if (sequence.getPrefixe().startsWith(draftPrefix))
      throw new AxelorException(
          sequence,
          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
          I18n.get("Sequence prefix cannot start with the draft prefix '%s'."),
          draftPrefix);
```